pom.xml

```diff
@@ -59,7 +59,7 @@
         <com.googlecode.gson.version>2.10.1</com.googlecode.gson.version>
         <com.h2database.version>2.2.224</com.h2database.version>
         <com.jcraft.jsch.version>0.1.55</com.jcraft.jsch.version>
-        <com.squareup.okhttp3.version>3.14.9</com.squareup.okhttp3.version>
+        <com.squareup.okhttp3.version>5.0.0-alpha.14</com.squareup.okhttp3.version>
```

Comment:
do we really need `alpha` version ? or vanilla 5.0.0 would work

Reply:
Switched to the latest stable version `4.12.0`